source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
+      :tabid: project-file
+
+      1. Open a project file.
+
+         To open a file in |vscode-short|, click on the file name in the
```

Comment:
(nit -- [click](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#c))
You can usually drop on. There are a couple instances. I'll just mark this one. 
```suggestion
         To open a file in |vscode-short|, click the file name in the
```

Reply:
Done!